pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -294,19 +294,21 @@ ul {
     display: flex;
     justify-content: start;
     flex-wrap: wrap;
+    gap: 1em 3em;
```

Comment:
Refactored the link list in the section under the TeachAI announcement to work w/ rtl languages (thanks @megcrenshaw!)
<img width="1052" alt="Screenshot 2023-04-25 at 9 06 30 AM" src="https://user-images.githubusercontent.com/9256643/234337202-f756a8b8-ff2a-449a-8f9f-46083d73c440.png">

Reply:
Hurray! Looks great! This has been a huge learning curve for me too 📖 🥳